.env

```diff
@@ -0,0 +1,13 @@
+# NOTE: These have to be prefixed REACT_APP
+# @see https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables
+
+# URL to Ahti GraphQL API
+REACT_APP_AHTI_GRAPHQL_API_URI=https://ahti-api.test.kuva.hel.ninja/graphql
```

Comment:
Since currently we only have the one test environment this should make the API work.

Once we get the production environment (which would use the production API) I think we would need to have this as a docker image build argument. Which would mean having the build time argument defined in the `Dockerfile` like [this](https://github.com/City-of-Helsinki/kukkuu-ui/blob/develop/Dockerfile#L57) and then assigning it a value in `.gitlab-ci.yml` like [this](https://github.com/City-of-Helsinki/kukkuu-ui/blob/develop/.gitlab-ci.yml#L25)

But as said this should also work for the time being.

Reply:
Will do for the new repo https://github.com/city-of-helsinki/ahti-ui-new, possibly as part of https://helsinkisolutionoffice.atlassian.net/browse/AHTI-169?atlOrigin=eyJpIjoiNzY3MGRjOGUxZWQ1NDVlYWIzZjcwY2I3ZDM2MjBiNWMiLCJwIjoiaiJ9.